### DISCUSSION TITLE
Loop not working ?  How to turn on looping ?

### DISCUSSION THREAD

**Discussion by vmars316** [smplayer-dev/smplayer#1079] [Q&A]
Greetings:
Pls , how to turn on looping ?
I tried this but no go :
Is it different sound files vs video ?

To loop a video forever in SMPlayer, you can use the Advanced Options1
. Here’s how:

Open SMPlayer and go to Options > Preferences1
.
Navigate to the Advanced tab1
.
In the Options for MPlayer/mpv field, type loop=1 and press Enter1 .
.

**Comment by smplayer-dev** [comment]
Menu Play -> A-B section -> Repeat.

**Reply by vmars316** [reply]
AH yes Thank you...

**Reply by James-LSK** [reply]
How do you repeat as default on every file? The "Advanced -> Run the following actions every time a file is opened" filled with -loop does not work anymore.

**Reply by smplayer-dev** [reply]
Use --loop-file=inf
